src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+    private int index = random.nextInt(Color.values().length);
```

Comment:
Did you test your solution? Did the getRandomColor truly return random color?

This field would be the same for the same object, so make it a local variable 
```suggestion
    private int index = random.nextInt(Color.values().length);
```